README.md

```diff
@@ -809,7 +857,7 @@ colord("#646464").alpha(0.5).toLchString(); // "lch(42.37% 0 0 / 0.5)"
 <details>
   <summary><b><code>mix</code> (Color mixing)</b> <i>0.9 KB</i></summary>
```

Comment:
Could I ask you to run `npm run size` locally and update the size here? 🙏

Reply:
Sure, will do!